recipes/httptools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "httptools" %}
+{% set version = "0.0.9" %}
+{% set sha256 = "f47f9870f19e3488e8def0898c46f87d36e5d59e90eb77d01453a6747cdab9f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - httptools
+
+about:
+  home: https://github.com/MagicStack/httptools
+  summary: "Fast HTTP parser"
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - 1st1
```

Comment:
@1st1, are you interested in [being a co-maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the conda-forge package of httptools?